### PR TITLE
Fix forced powering off

### PR DESCRIPTION
**Pull request by nest1ing**

### CODE REVIEW

File: commands/host.vegman [hunk 1]
```diff
@@ -98,12 +98,11 @@ function cmd_power_off {
...
 
   if [[ -n "${force}" ]]; then
-    ipmitool chassis power off
-    echo "Chassis power is switched off, host is down."
+    ipmitool power off
   else
     ipmitool power soft
-    echo "Shutdown signal sent to host, waiting to power the system off."
   fi
+  echo "Shutdown signal sent to host, waiting to power the system off."
```
Comment (artemsen) [inline commands/host.vegman]: `ipmitool power off` will turn the host off an any case. Why "waiting"?

Comment (AlexanderAmelkin) [reply commands/host.vegman]: Yup. Besides, nothing in the code is actually 'waiting'.

**Review by AlexanderAmelkin — APPROVED** [review]

File: commands/host.vegman [hunk 2]
```diff
@@ -65,8 +65,7 @@ function cmd_power {
 # Turn the host on
 function cmd_power_on {
   expect_noarg "$@"
-  ipmitool power on
-  echo "Chassis power is switched on, host is booting up."
+  hostpwrctl on
```
Comment (AlexanderAmelkin) [inline commands/host.vegman]: Need `exec -a` for `hostpwrctl`'s help to work properly.
The `hostpwrctl` command itself shouldn't be available to the users, so they need to get help in the form of a CLI command.

**Review by AlexanderAmelkin — CHANGES_REQUESTED** [review]

**Review by nest1ing — COMMENTED** [review]
Looks good to me, but github doesn't allow me to approve it as I'm an author of the PR.

**Comment by nest1ing** [comment]
And this PR requires https://github.com/YADRO-KNS/obmc-yadro-hostpwrctl/pull/1

**Pull request merged**